### PR TITLE
Add Fx News one-click-subscribe page (Fixes #15142)

### LINKED ISSUE #15142: 1 click subscribe LP for Firefox News
Hi. This is similar to a request we received a few months ago (https://github.com/mozilla/bedrock/issues/14823).

This time, they would like the same flow with a new landing page that allows prior subscribers of the _Security and Privacy (S&P)_ newsletter to join the _Firefox News_ newsletter in one-click. 

They are requesting this to be in FR and DE.

The design and layout can be the same as the one in the previous ticket referenced. 

The timeline is flexible. 

The copy docs with localized strings are below: 
[- DE ](https://docs.google.com/spreadsheets/d/1mM1H65Ph6dFAQCkQsv-LhWAsHgvnOv50vUPiaF4P0To/edit?gid=2048776457#gid=2048776457)
[- FR](https://docs.google.com/spreadsheets/d/1__IbP_pPrGgGa-Ac7ETj0NmAzRwiCqRLv0X250R047M/edit?gid=844373989#gid=844373989)

Thank you. Please let me know if there are any questions or concerns.

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds one-click confirmation landing page for Firefox newsletter

## Issue / Bugzilla link

#15142

## Testing

First get your newsletter token by following these steps:

1. Open https://www.mozilla.org/en-US/newsletter/recovery/ and enter your email.
2. Find the recovery email, and click the link that opens the email preference center.
3. Open Dev Tools -> Storage -> Cookies and find the cookie called `nl-token`. Copy it's value (which should be a long alpha-numeric string).
4. Whilst you're here, make sure you are not subscribed to the Firefox newsletter (unsubscribe if you are).

Next, open the confirmation page using your token in the URL:

1. Open: `http://localhost:8000/en-US/newsletter/firefox/confirm/{INSERT_YOUR_TOKEN_HERE}`
2. When the page loads, click "Confirm".
3. Wait to see a "thank you" message and then you should now be subscribed to Firefox news.
4. Finally, confirm the subscription has worked by going back to the newsletter management page, and you should hopefully see Firefox News is checked in the list.

### CODE REVIEW

**Comment by alexgibson** [comment]
This is ready for code review, but marking as do-not-merge pending approval of final copy.

**Comment by alexgibson** [comment]
OK, this is finally approved and good to merge once it's been reviewed.

**Review by craigcook — APPROVED** [review]
📫

**Pull request merged**
